### PR TITLE
Clarify assertField... usage description

### PR DESCRIPTION
**Pull request by jonataswalker**

### CODE REVIEW

**Comment by istr** [comment]
Basically ok to document the separate functions. However, `assertField` marshals, so the description is not *wrong*. Please add blocks to the documentation that show usage of `assertField`, too for each type of selector so we don't lose the information how `assertField` marshals, and demonstrate that we could explicitly marshal on our own. Note, that the `assertFieldName` function does not exist in master yet.

**Comment by jonataswalker** [comment]
Well ok, closing then.

**Comment by istr** [comment]
Sorry... Misunderstanding. This PR is welcome, but I think it needs an update: just do not *replace* the existing doc but *amend* it. Please re-open if you are willing to adapt it. Thank you.

**Comment by jonataswalker** [comment]
Reopened @istr, let me know what to do from here.

File: docs/modules/tester.rst [hunk 1]
```diff
@@ -203,10 +203,10 @@ The `options` parameter allows to set the options to use with
 
 Asserts that a given form field has the provided value::
 
-    casper.test.begin('assertField() tests', 1, function(test) {
```
Comment (istr) [inline docs/modules/tester.rst]: keep this line

File: docs/modules/tester.rst [hunk 2]
```diff
@@ -203,10 +203,10 @@ The `options` parameter allows to set the options to use with
...
+    casper.test.begin('assertFieldName() tests', 1, function(test) {
```
Comment (istr) [inline docs/modules/tester.rst]: prefix with
// TODO:

File: docs/modules/tester.rst [hunk 3]
```diff
@@ -203,10 +203,10 @@ The `options` parameter allows to set the options to use with
...
         casper.start('http://www.google.fr/', function() {
             this.fill('form[name="gs"]', { q: 'plop' }, false);
-            test.assertField('q', 'plop', 'did not plop', {formSelector: 'plopper'});
```
Comment (istr) [inline docs/modules/tester.rst]: keep this line

File: docs/modules/tester.rst [hunk 4]
```diff
@@ -203,10 +203,10 @@ The `options` parameter allows to set the options to use with
...
+            test.assertFieldName('q', 'plop', 'did not plop', {formSelector: 'plopper'});
```
Comment (istr) [inline docs/modules/tester.rst]: prefix with
// TODO:

File: docs/modules/tester.rst [hunk 5]
```diff
@@ -221,10 +221,10 @@ Asserts that a given form field has the provided value::
 
 Asserts that a given form field has the provided value given a CSS selector::
 
-    casper.test.begin('assertField() tests', 1, function(test) {
+    casper.test.begin('assertFieldCSS() tests', 1, function(test) {
```
Comment (istr) [inline docs/modules/tester.rst]: set test number from 1 to 2

File: docs/modules/tester.rst [hunk 6]
```diff
@@ -221,10 +221,10 @@ Asserts that a given form field has the provided value::
...
         casper.start('http://www.google.fr/', function() {
             this.fill('form[name="gs"]', { q: 'plop' }, false);
-            test.assertField('q', 'plop', 'did not plop', 'input.plop');
```
Comment (istr) [inline docs/modules/tester.rst]: keep this line

File: docs/modules/tester.rst [hunk 7]
```diff
@@ -239,10 +239,10 @@ Asserts that a given form field has the provided value given a CSS selector::
 
 Asserts that a given form field has the provided value given a XPath selector::
 
-    casper.test.begin('assertField() tests', 1, function(test) {
+    casper.test.begin('assertFieldXPath() tests', 1, function(test) {
```
Comment (istr) [inline docs/modules/tester.rst]: set test number from 1 to 2

File: docs/modules/tester.rst [hunk 8]
```diff
@@ -239,10 +239,10 @@ Asserts that a given form field has the provided value given a CSS selector::
...
         casper.start('http://www.google.fr/', function() {
             this.fill('form[name="gs"]', { q: 'plop' }, false);
-            test.assertField('q', 'plop', 'did not plop', '/html/body/form[0]/input[1]');
```
Comment (istr) [inline docs/modules/tester.rst]: keep this line

**Comment by istr** [comment]
Thank you! Added some comments.

File: docs/modules/tester.rst (hunk 1 above)
Comment (jonataswalker) [inline docs/modules/tester.rst]: Isn't it `assertFieldName` description method?

File: docs/modules/tester.rst (hunk 3 above)
Comment (jonataswalker) [inline docs/modules/tester.rst]: Also here, isn't it `assertFieldName` usage example?

File: docs/modules/tester.rst (hunk 5 above)
Comment (jonataswalker) [inline docs/modules/tester.rst]: Sorry, but where is the second test?

File: docs/modules/tester.rst (hunk 6 above)
Comment (jonataswalker) [inline docs/modules/tester.rst]: Also here, isn't it `assertFieldCSS` usage example?

File: docs/modules/tester.rst (hunk 7 above)
Comment (jonataswalker) [inline docs/modules/tester.rst]: Same here, there's just a single assert.

File: docs/modules/tester.rst (hunk 8 above)
Comment (jonataswalker) [inline docs/modules/tester.rst]: And here is `assertFieldXPath` usage example.

**Comment by jonataswalker** [comment]
Hey @istr sorry the delay.

I also added some comments.

**Comment by jefleponot** [comment]
@jonataswalker , do you still want to merge your PR ?

**Comment by jonataswalker** [comment]
Yes @jefleponot

**Pull request merged**